geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -211,7 +231,16 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType
       val remoteFilters = if (remote) { ecql.map { filter =>
         new JSimpleFeatureFilter(sft, filter)
       }.toSeq } else { Nil }
-      ScanConfig(remoteFilters, toFeatures)
+
+      val coprocessor: Option[Coprocessor] = {
+        if (hints.isDensityQuery) {
+          Some(new KryoLazyDensityCoprocessor)
```

Comment:
coprocessors aren't going to work on bigtable I assume... think this should be reverted